### PR TITLE
Phonons docs

### PR DESCRIPTION
**Pull request by antoine-levitt**

### CODE REVIEW

**Comment by antoine-levitt** [comment]
@mfherbst not up for review yet, still ongoing with Etienne

**Comment by antoine-levitt** [comment]
Didn't we say we could get rid of memoization by doing get_kpoint_from_equivalent(kcoord, equiv_kpoint) and just translating the G vectors from the equivalent kpoint?

**Comment by antoine-levitt** [comment]
We should be able to deduce it from the one of the old kpoint, no? I'm not super concerned about it being very efficient, worse case we just recompute everything...

File: src/transfer.jl [hunk 1]
```diff
@@ -202,60 +190,47 @@ function find_equivalent_kpt(basis::PlaneWaveBasis{T}, kcoord, spin; tol=sqrt(ep
...
-`kpoints[ik].coordinate + q` is equivalent to `kpoints[indices[ik]].coordinate`.
+Construct a k-point from its equivalent (in the meaning of [`find_equivalent_kpt`](@ref)).
+Faster than computing it from scratch.
+"""
+function construct_from_equivalent_kpt(basis, equivalent_kpt, coordinate, ΔG)
+    linear = LinearIndices(basis.fft_size)
+    # Mapping is the same as if created from scratch, although it is not ordered.
+    mapping = map(CartesianIndices(basis.fft_size)[equivalent_kpt.mapping]) do G
+        linear[CartesianIndex(mod1.(Tuple(G + CartesianIndex(ΔG...)), basis.fft_size))]
+    end
```
[the remark this answers is not in the document]
Comment (antoine-levitt) [reply src/transfer.jl]: What do you mean we can't use it? Everything looks fine and consistent to me, we don't assume ordering anywhere? Maybe just add to Kpoint's docstring that it's not assumed to be ordered? (we do assume ordering of G_vectors(basis) however)

**Comment by antoine-levitt** [comment]
Good to merge for me when you want @epolack

**Comment by mfherbst** [comment]
Nice, looks good.

**Pull request merged**